lib/jsxgettext.js

```diff
@@ -186,8 +191,14 @@ function parse(sources, options) {
             }
           };
           if( arg.constructor === Array ) {
-              translations[str].msgid_plural = extractStr(arg[1]);
-              translations[str].msgstr = ['', ''];
+              //If this is context string
+              if (arg.length === 3) {
+                  translations[str].msgctxt = extractStr(arg[1]);
+              //this is plural
+              } else {
+                translations[str].msgid_plural = extractStr(arg[1]);
+                translations[str].msgstr = ['', ''];
+              }
```

Comment:
Another option is for function getTranslatable to return object instead of array. And in object you would have context, singular, plural, domain keys values would be the same objects that are in array. That way you can have context with plurals and other combinations. IMHO.